src/test/java/com/seatapp/SeatAppApplicationTests.java

```diff
@@ -4,6 +4,7 @@
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
+@SuppressWarnings("PMD")
```

Comment:
also a suppression that needs to go?

Reply:
Yes same thing as the other one... Is now removed.